src/__tests__/regression/test-cases/read-state.json

```diff
@@ -86,7 +86,6 @@
   "2caOW6ol9T8LHCMO8tVAx4GHRwv1q3fFc79KzKOtoww",
   "HdZBZa0GfOEUYCubwvoSyxGUUgPmgy7RJb5l77T21bE",
   "7Dp5r-UpZLDvHqsDbZbqWhCBwbYdJMKBuC3tFC-FF7U",
-  "YLVpmhSq5JmLltfg6R-5fL04rIRPrlSU22f6RQ6VyYE",
```

Comment:
removing. Slowing down the whole SDK just because of this one, abandoned contract, is not very wise :-)
original issue: https://github.com/ArweaveTeam/SmartWeave/pull/92